crates/core/src/rpc/surfnet_cheatcodes.rs

```diff
@@ -711,6 +711,9 @@ pub trait SurfnetCheatcodes {
     /// ```
     #[rpc(meta, name = "surfnet_resumeClock")]
     fn resume_clock(&self, meta: Self::Metadata) -> Result<EpochInfo>;
+
+    #[rpc(meta, name = "surfnet_resetAccount")]
+    fn reset_account(&self, meta: Self::Metadata, pubkey_str: String) -> Result<RpcResponse<()>>;
```

Comment:
Currently we aren't allowing the user to pass any config, can we add that?


```
    fn reset_account(&self, meta: Self::Metadata, pubkey_str: String, config: ResetAccountConfig) -> Result<RpcResponse<()>>;
```

and in our types something like
```rust
pub struct ResetAccountConfig {
    pub cascade_to_owned: Option<bool>
}
```